src/Forte.Optimizely.ContentUsage/Frontend/Views/ContentTypeUsageView.scss

```
@@ -1,4 +1,20 @@
 .forte-optimizely-content-usage-spinner__center {
     display: flex;
     justify-content: center;
-}
\ No newline at end of file
+}
+
+.forte-optimizely-content-usage-page-url {
+    padding: 0 8px;
+    width: 100%;
+    &:hover {
+        background-color: var(--grey-50);
+    }
+}
+
+.oui-disclose.is-active{
+    .oui-disclose__content {
```

I think you missed `&` here:
```suggestion
    & .oui-disclose__content {
```